sound/soc/codecs/rt1011.c

```diff
@@ -1801,7 +1801,10 @@ static int rt1011_set_tdm_slot(struct snd_soc_dai *dai,
 	struct snd_soc_dapm_context *dapm =
 		snd_soc_component_get_dapm(component);
 	unsigned int val = 0, tdm_en = 0, rx_slotnum, tx_slotnum;
-	int ret = 0, first_bit, last_bit;
+	int ret = 0;
+	unsigned int first_bit, last_bit;
```

Comment:
@fredoh9 do we need to change the type for these? I know find_nex_bit() returns an unsigned long. So if we're chaning it it should be a separate patch and unsigned long right?

Reply:
This is not required change. I should just drop it. Will update again.